corehq/apps/geospatial/routing_solvers/pulp.py

```diff
@@ -33,52 +33,94 @@ def calculate_distance_matrix(self, config):
 
         return distance_matrix
 
-    def solve(self, config, print_solution=False):
+    def solve(self, config):
         costs = self.calculate_distance_matrix(config)
+
+        if not costs:
+            # Infeasible solution
+            return None, None
```

Comment:
nit: maybe consider raising exceptions in case of infeasible solution or solution not found. Feel like could be more explicit.

Reply:
I was initially thinking about raising an error, but then I thought it's not actually an error, it's just that there's no solution, hence returning a `None`.